src/views/SplitPanelResizer.js

```diff
@@ -20,14 +20,15 @@ export default Marionette.View.extend({
 
     onRender() {
         Core.services.UIService.undraggable({ el: this.el });
+        this.__debounceOnResize = _.debounce(this.__handleWindowResize, 50);
+        this.listenTo(Core.services.GlobalEventService, 'window:resize', this.__debounceOnResize);
         if (this.options.orientation === splitViewTypes.HORIZONTAL) {
-            const originalPanel1Height = this.firstPanel.el.getBoundingClientRect().height;
-
-            this.$el.css('left', '');
-            this.$el.css('top', `${originalPanel1Height + this.firstPanel.el.offsetTop}px`);
+            const originalPanelHeight = this.firstPanel.el.getBoundingClientRect().height;
+            this.splitRatio = originalPanelHeight / window.innerHeight;
+            this.topOffset = this.firstPanel.el.offsetTop;
```

Comment:
top from this.firstPanel.el.getBoundingClientRect()

Reply:
done